hack/ginkgo-e2e.sh

```diff
@@ -89,5 +89,7 @@ fi
 "${e2e}" "${auth_config[@]:+${auth_config[@]}}" \
   --host="https://${KUBE_MASTER_IP-}" \
   --provider="${KUBERNETES_PROVIDER}" \
+  --ginkgo.v \
+  ${E2E_FOCUS+"--ginkgo.focus=${E2E_FOCUS}"} \
```

Comment:
Please don't use `--ginkgo.focus`, instead use `-t` or `--test` which matches a whole test name. In fact, goe2e.sh already takes $@ arguments so you could simply pass them through here, you don't really need an exported environment variable here.